contracts/DecentAutonomousAdmin.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {IHats} from "./interfaces/hats/full/IHats.sol";
+import {IHatsElectionEligibility} from "./interfaces/hats/full/IHatsElectionEligibility.sol";
+import {ISablierV2Lockup} from "./interfaces/sablier/full/ISablierV2Lockup.sol";
+
+contract DecentAutonomousAdmin {
+    string public constant NAME = "DecentAutonomousAdmin";
+    string public version_ = "0.1.0";
```

Comment:
But actually...

For this contract, which is a "long standing" contract (aka not an ephemeral contract in the context of a proposal execution that we add/remove as a module to execute specific logic), I'd like to see ERC165 support built in. Will be good for having a programmatic way to determine contract functionality when this contract will inevitably be upgraded in the future, and we may have DAOs with different versions of this contract installed on their Admin role.

I think the ERC165 support should replace the (custom, non-idiomatic) `NAME` and `version_` variables.

Reply:
Just saw these comments over the weekend. Let me get this done